### ISSUE TITLE
HUD wont open at all.

### ISSUE THREAD

**Issue by karastormchaser** [OpenEmu/OpenEmu#3859]
I used to use OpenEmu for a long while to play ROMs. The HUD would always open and I would be able to use it freely to add my cheats in where I needed. I use NES, Game Gear, Sega Genesis, and Sony PlayStation. Today I decided to boot my Mac up for the first time in a year or so to play some games and it automatically upgraded me to 2.0.8 (016-g31a4fbf-Release), and now the HUD wont appear at all unless I open my notes file, then close it in order to "force" the HUD to show up. I've tried to make sure that the "Always Hide HUD" open is unchecked and it usually is. When I encountered this problem first today, I was using MacOS Sierra. I figured if maybe I upgraded to Mojave, it would work. It has not. I also noticed that when I checked my cores, NEStopia core disksys.rom has an orange check mark by it. I'm not sure if that's part of the problem.

I'm not really a programmer or anything, I just stumbled across the problem and have been trying to find help to no avail.

**Comment by karastormchaser** [comment]
> > and now the HUD wont appear at all unless I open my notes file, then close it in order to "force" the HUD to show up
> 
> You're going to have to be more clear than that. This isn't something anyone can normally replicate, so what do you mean by "notes file"?



Notes file = another app that is literally called Notes. I have all of my game genie codes on there that i normally use in the HUD.

**Comment by clobber** [comment]
Cannot replicate the in-game HUD bar not showing when Notes app is open or closed.

**Comment by karastormchaser** [comment]
> Cannot replicate the in-game HUD bar not showing when Notes app is open or closed.


I tried to uninstall the app and reinstall it, but it doesnt seem to want to follow the normal thing of "throw it in the trash and that uninstalls a program on mac". I redownloaded it, and now its showing me that the program is a unix executable thats opened from the internet. No .dmg files to drag and drop like usual. 

im really confused by this whole thing.

**Comment by robotoad** [comment]
The download button on the [official website](http://openemu.org) links to a zip file of the app ([version 2.0.8](https://github.com/OpenEmu/OpenEmu/releases/download/v2.0.8/OpenEmu_2.0.8.zip) as of this post). There should be a zipper on the icon. If the file didn't download correctly, please try again.

**Comment by clobber** [comment]
>normal thing of "throw it in the trash and that uninstalls a program on mac"

Deleting the app doesn't mean all the previous settings get wiped out. That isn't true for any Mac app, unless they ship a separate uninstaller program or script...

 >showing me that the program is a unix executable thats opened from the internet.

That isn't what it's showing you. The application is unsigned and that is the standard message one would see. More details here, which was linked directly on our website... https://support.apple.com/kb/PH25088?locale=en_US

>No .dmg files to drag and drop like usual.

You seem very confused. We do not package the application in a dmg installer. Never have.

**Comment by karastormchaser** [comment]
> > normal thing of "throw it in the trash and that uninstalls a program on mac"
> 
> Deleting the app doesn't mean all the previous settings get wiped out. That isn't true for any Mac app, unless they ship a separate uninstaller program or script...
> 
> > showing me that the program is a unix executable thats opened from the internet.
> 
> That isn't what it's showing you. The application is unsigned and that is the standard message one would see. More details here, which was linked directly on our website... https://support.apple.com/kb/PH25088?locale=en_US
> 
> > No .dmg files to drag and drop like usual.
> 
> You seem very confused. We do not package the application in a dmg installer. Never have.


You're right. I am very confused as to why my HUD isnt showing up on the program any longer. And so far I haven't really gotten much help from this at all. Please don't condescend to me. I don't appreciate it.


As I said: I am running the newest OpenEmu with the newest MacOS Mojave. my HUD will not show up anymore when I go to move my mouse as it used to do. These steps I have described is what I have seen.

**Comment by robotoad** [comment]
You said the Always Hide HUD option is "usually" unchecked. To be clear, when the HUD bar isn't showing up, the Always Hide HUD option is definitely unchecked? Is your gameplay running in the main window or in a popout window?

**Comment by karastormchaser** [comment]
> You said the Always Hide HUD option is "usually" unchecked. To be clear, when the HUD bar isn't showing up, the Always Hide HUD option is definitely unchecked? Is your gameplay running in the main window or in a popout window?


Its been unchecked the last few times I've seen it. I've tried running it in the main window and in an popout window as well. When I played it in the past, it would be in the main window and the HUD would work just fine. There was a single time today when I didnt have a game booted up and I looked at that option, it was all greyed out and it was checked. But when I loaded a game, the open became clickable and it was unchecked.
